src/downloadPubchemInfo.py

```diff
@@ -0,0 +1,108 @@
+# Porthmeus
+# 15.02.23
+
+
+'''
+Get a list of metabolite names, search pubchem for it and retrieve an InChI/Smile string from it.
+'''
+
+
+import pubchempy as pcp
+import pandas as pd
+import numpy as np
+import time
+import sys
+
+# global variable for the atoms list
+atoms_list = ['H', 'He', 'Li', 'Be', 'B', 'C', 'N', 'O', 'F', 'Ne', 'Na', 'Mg', 'Al', 'Si', 'P', 'S', 'Cl', 'Ar', 'K', 'Ca', 'Sc', 'Ti', 'V', 'Cr', 'Mn', 'Fe', 'Co', 'Ni', 'Cu', 'Zn', 'Ga', 'Ge', 'As', 'Se', 'Br', 'Kr', 'Rb', 'Sr', 'Y', 'Zr', 'Nb', 'Mo', 'Tc', 'Ru', 'Rh', 'Pd', 'Ag', 'Cd', 'In', 'Sn', 'Sb', 'Te', 'I', 'Xe', 'Cs', 'Ba', 'La', 'Ce', 'Pr', 'Nd', 'Pm', 'Sm', 'Eu', 'Gd', 'Tb', 'Dy', 'Ho', 'Er', 'Tm', 'Yb', 'Lu', 'Hf', 'Ta', 'W', 'Re', 'Os', 'Ir', 'Pt', 'Au', 'Hg', 'Tl', 'Pb', 'Bi', 'Po', 'At', 'Rn', 'Fr', 'Ra', 'Ac', 'Th', 'Pa', 'U', 'Np', 'Pu', 'Am', 'Cm', 'Bk', 'Cf', 'Es', 'Fm', 'Md', 'No', 'Lr', 'Rf', 'Db', 'Sg', 'Bh', 'Hs', 'Mt', 'Ds', 'Rg', 'Cn', 'Nh', 'Fl', 'Mc', 'Lv', 'Ts', 'Og', 'R']
+
+
+def downloadElements() -> pd.DataFrame:
+    '''
+    Download the elements table from pubchem and return it as a data frame
+    '''
+
+    # download the element list from pubchem
+    elements = pd.read_csv("https://pubchem.ncbi.nlm.nih.gov/rest/pug/periodictable/CSV?response_type=save&response_basename=PubChemElements_all")
+    return(elements)
+
+def looksLikeSumFormula(string:str, atoms:list = atoms_list) -> bool:
```

Comment:
Please include the type of the List i.e. it should be list[str]
See https://docs.python.org/3/library/typing.html for more Informatiom